local-setup-testing/test/main.test.ts

```diff
@@ -2,39 +2,72 @@ import { expect } from 'chai';
 import { Wallet, Contract, utils } from 'zksync-web3';
 import * as hre from 'hardhat';
 import { ethers } from 'ethers';
-import { Deployer } from "@matterlabs/hardhat-zksync-deploy";
+import { Deployer } from '@matterlabs/hardhat-zksync-deploy';
 
-const RICH_WALLET_PK = '0x7726827caac94a7f9e1b160f7ea819f172f7b6f9d2a97f992c38edeab82d4110';
+const RICH_WALLET_PK =
+  '0x7726827caac94a7f9e1b160f7ea819f172f7b6f9d2a97f992c38edeab82d4110';
 
-async function deployGreeter(deployer: Deployer): Promise<Contract> {
-    const artifact = await deployer.loadArtifact("Greeter");
-    return await deployer.deploy(artifact, ["Hi"]);
-}
+let greeter: Contract;
+
+async function deployGreeter(deployer: Deployer) {
+  try {
+    console.log('Deploying contract');
+    const artifact = await deployer.loadArtifact('Greeter');
 
-describe("Greeter", function () {
-    let deployer: Deployer;
+    // return await deployer.deploy(artifact, ['Hi']);
```

Comment:
Commented code

Reply:
fixed in 070e48285b7b7f49847a2bb180adb57ca86b7468